### DISCUSSION TITLE
Runs before certain build folders are available

### DISCUSSION THREAD

**Discussion by gregg-cbs** [sapphi-red/vite-plugin-static-copy#49] [Q&A]
I have a nuxt3 project and I want to copy a static file into the `.output/server/` folder.

It seems the copy plugin runs before the build is complete:
![image](https://user-images.githubusercontent.com/57114335/217815689-745ca9d5-3a8d-40b0-bb80-0f848972ea9b.png)

Here is a video the plugin copying the file (orders.json) and then it gets removed as the build progresses:

https://user-images.githubusercontent.com/57114335/217817266-989e7dfb-1c7c-4d6b-9b28-2899531dfa7c.mp4

Is there any way around this?

```ts
// https://nuxt.com/docs/api/configuration/nuxt-config
import { viteStaticCopy } from 'vite-plugin-static-copy'
import { normalizePath } from 'vite'
import path from 'node:path'

export default defineNuxtConfig({
  pages: true,
  modules: ["nuxt-windicss"],
  vite: {
    plugins: [
      viteStaticCopy({
        targets: [
          {
            src: normalizePath(path.resolve(__dirname, 'data/orders.json')),
            dest: path.resolve(__dirname, '.output')
          }
        ]
      })
    ] 
  }
})

```

**Comment by sapphi-red** [comment]
I don't know about how vite plugins work with nuxt. But I guess you need to use this plugin when building the SSR bundle.

**Comment by 4KDA** [comment]
@gregg-cbs Have you found a solution?

**Reply by gregg-cbs** [reply]
yes. I moved over to Nextjs 😅
